src/jobs/ProcessBulkOperationData.php

```diff
@@ -114,6 +125,13 @@ protected function before(): void
         $bulkOperation->setStatus(BulkOperationStatus::Processing);
 
         Plugin::getInstance()->getBulkOperations()->saveBulkOperation($bulkOperation, false);
+
+        // Clear data if requested
+        if ($this->clearData === BulkOperationRecord::CLEAR_DATA_ALL) {
+            ShopifyData::updateAll(['stale' => true]);
+        } elseif ($this->clearData !== BulkOperationRecord::CLEAR_DATA_NONE) {
+            Plugin::getInstance()->getProducts()->markShopifyDataStaleByShopifyId($this->clearData);
```

Comment:
Is `$this->clearData` a Shopify ID?

Reply:
Correct it is either `all`, `none` or gid string. Documented it here: https://github.com/craftcms/shopify/pull/130/files#diff-962e5c9a236307f7191b2c5ef28ed371cdb54b1e187cec390760ef8e79196468R39-R45